tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<ul><li>You should mention that the value of C3 is depends on the device.</li><li>Both your basis states are named |0&gt;. Also, weren't we using "up" and "down" here?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yes we use 'up' and 'down' but isn't it equivalent to use |0> and |1>?

Reply:
It is equivalent (it's just a matter of naming after all) but "up" and "down" is probably clearer. |0> and |1> can be ambiguous

Reply:
|0> and |1> are standard in quantum computing whereas 'up' and 'down' are completely made by us

Reply:
Sure, though you're not using this for digital quantum computation and what you decide to call |0> and |1> is also made up by you. My usual issue with |0> and |1> is that the convention in the Bloch sphere is to have |0> on the north pole and |1> on the south. Notice how here you are actually defining |down> = |0> and |up> = |1>,  so "down" would be on the north pole and "up" on the south (which is completely counter-intuitive, in my opinion). 

The reason behind my suggestion here was to have an unequivocal definition, which would also distinguish these states from the ones used in the Ising mode. If you insist on using these, I won't force you to change it, we can always do it later if necessary.

Reply:
I can see you are not keen to change it, so I'll let the nomenclature stay like this